setup.py

```diff
@@ -24,6 +24,7 @@
     'psutil',
     'pyprind',
     'python-dateutil',
+    'pytorch-cpu torchvision-cpu -c pytorch',
```

Comment:
is there a reason these each can't be different entries?

Reply:
No I will separate them